Project.toml

```diff
@@ -23,6 +23,7 @@ ForwardDiff = "f6369f11-7733-5829-9624-2563aa707210"
 IterativeSolvers = "42fd0dbc-a981-5370-80f2-aaf504508153"
 KrylovMethods = "9a2cd570-f05c-5dc1-9209-93ad6f5727f7"
 NBInclude = "0db19996-df87-5ea3-a455-e3a50d440464"
+PkgBenchmark = "32113eaa-f34f-5b0d-bd6c-c81e245fc73d"
```

Comment:
Should probably be in `benchmark/Project.toml`?

Reply:
Sorry to ask, but how can I ensure that this subproject uses the correct Ferrite branch as dependency? `add ..` seems to fix the branch
```
...
[c061ca5d] + Ferrite v0.3.8 `..#do/performance-regression-tests`).
...
```